src/misc/cache/lv_cache.c

```diff
@@ -82,6 +82,12 @@ lv_cache_entry_t * lv_cache_acquire(lv_cache_t * cache, const void * key, void *
     LV_ASSERT_NULL(key);
 
     lv_mutex_lock(&cache->lock);
+
+    if(cache->size == 0) {
```

Comment:
This `if` statement could go above `lv_mutex_lock()`, and then return immediately, and no need to call `lv_mutex_lock()`.

```suggestion
    if(cache->size == 0) {
          return NULL;
    }
    
    lv_mutex_lock(&cache->lock);
    ....
```

Reply:
I think there may be some data competition risks here. When the same cache adds cache from another thread, the whole cache is locked. At this time, the size of the cache is in the middle state, and it is not sure whether it is really 0. After the process releases the mutex lock, it can accurately get the size of the cache. Of course, I'm not sure if this demand is necessary 😄